OpenRA.Mods.Common/Scripting/Global/UtilsGlobal.cs

```diff
@@ -106,7 +103,8 @@ public int RandomInteger(int low, int high)
 		[Desc("Returns the ticks formatted to HH:MM:SS.")]
 		public string FormatTime(int ticks, bool leadingMinuteZero = true)
 		{
-			return WidgetUtils.FormatTime(ticks, leadingMinuteZero, world.Timestep);
```

Comment:
Sorry for misleading you: this needs to be 40 (1000 / 25) instead of 25.  Timestep measures the number of milliseconds per tick, not ticks per second.